contracts/base/Initializable.sol

```diff
@@ -43,6 +43,10 @@ contract Initializable {
 
     /** External Functions */
 
+    function initialized() external view returns (bool) {
```

Comment:
Is there a reason to make this function? And not just make the variable public?

Reply:
it gets conflict with modifier isInitialized (that's why the variable is bool private _isInitialized).